webv2/table/update_table_schema.go

```diff
@@ -96,6 +98,14 @@ func UpdateTableSchema(w http.ResponseWriter, r *http.Request) {
 
 		if v.Rename != "" && v.Rename != conv.SpSchema[tableId].ColDefs[colId].Name {
 
+			oldName := conv.SrcSchema[tableId].ColDefs[colId].Name
```

Comment:
Same as above

Reply:
added regular expression to match the exact column name.